OP2Internal.vcxproj

```diff
@@ -45,12 +45,34 @@
       <DebugInformationFormat>ProgramDatabase</DebugInformationFormat>
       <Optimization>Disabled</Optimization>
     </ClCompile>
+    <CustomBuild>
+      <Command>"C:\Program Files (x86)\NASM\nasm" -fwin32 "%(FullPath)" -o %(Filename).obj</Command>
+    </CustomBuild>
+    <CustomBuild>
+      <Message>Build ASM File</Message>
+    </CustomBuild>
+    <CustomBuild>
+      <TreatOutputAsContent>true</TreatOutputAsContent>
+      <Outputs>%(Filename).obj</Outputs>
+    </CustomBuild>
+  </ItemDefinitionGroup>
+  <ItemDefinitionGroup Condition="'$(Configuration)|$(Platform)'=='Release|Win32'">
+    <CustomBuild>
+      <Command>"C:\Program Files (x86)\NASM\nasm" -fwin32 "%(FullPath)" -o %(Filename).obj</Command>
+      <Message>Build ASM File</Message>
+      <TreatOutputAsContent>true</TreatOutputAsContent>
+      <Outputs>%(Filename).obj</Outputs>
+    </CustomBuild>
```

Comment:
This looks like a duplicate of the above, but all under a single `CustomBuild` tag, like I expected.